src/auth-service/controllers/create-user.js

```diff
@@ -281,6 +282,83 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      const { query, body } = req;
+      let { tenant } = query;
```

Comment:
I am not sure about this **tenant** query parammter. Is this being utilised anywhere?

If it is being utilised, we shall alzo need to do input validation for it accordingly.

Thanks!

Reply:
Thanks @Baalmart Let me remove it.